From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Fri, 14 Sep 2018 16:39:40 +0200
Subject: [PATCH] docs: Stop using polkit / pkexec for grub-boot-success.timer
 / service

We also want to call grub2-set-bootflag under gdm and pkexec does not
work under gdm because the gdm user has /sbin/nologin as shell.

So instead we are going to install grub2-set-bootflag as suid root,
grub2-set-bootflag was written with this usage in mind, so is safe
to be made suid root.

Signed-off-by: Hans de Goede <hdegoede@redhat.com>
[rharwood: rebase fuzz around container]
Signed-off-by: Robbie Harwood <rharwood@redhat.com>
---
 docs/grub-boot-success.timer | 1 -
 1 file changed, 1 deletion(-)

diff --git a/docs/grub-boot-success.timer b/docs/grub-boot-success.timer
index 51d1eb238..406f17200 100644
--- a/docs/grub-boot-success.timer
+++ b/docs/grub-boot-success.timer
@@ -2,7 +2,6 @@
 Description=Mark boot as successful after the user session has run 2 minutes
 ConditionUser=!@system
 ConditionVirtualization=!container
-ConditionPathExists=/usr/bin/pkexec
 
 [Timer]
 OnActiveSec=2min
